lib/msf/core/session_compatibility.rb

```diff
@@ -188,7 +188,7 @@ def session_incompatibility_reasons(sess_or_sid)
 
       # Can't be compatible if it's the wrong type
       if session_types && !session_types.include?(s.type)
-        issues << "incompatible session type: #{s.type}"
+        issues << "incompatible session type: #{s.type}. This module works with: #{session_types.join(', ')}."
```

Comment:
Not sure if we like these capitalised maybe?

Reply:
Not a blocker for me :+1: